src/classes/CRLP_RollupAccSoftCredit_TEST.cls

```diff
@@ -0,0 +1,7 @@
+/**
+ * Created by bbreisnes on 2018-07-31.
+ */
+
+public with sharing class CRLP_RollupAccSoftCredit_TEST {
```

Comment:
This should be `private' without a sharing declaration to match the format of all the other CRLP unit tests. Generally there isn't a reason for unit tests to be public, and "with sharing" doesn't necessarily specify anything here because the methods cannot be called by other classes that might use "without sharing".

Reply:
yep sorry, this was just a placeholder!! it'll go in correctly on tuesday.